docs/User-Research/Overview of Research Activities.md

```diff
@@ -180,7 +179,7 @@ Internally, we also wanted to:
 
 **How we used this research**
 
-- We updated [our stakeholder map](https://github.com/HHS/TANF-app/wiki/Product-Strategy#project-stakeholders) and [user personas](https://github.com/HHS/TANF-app/wiki/User-types).
```

Comment:
Could you update these wiki links to be docs links?

Reply:
@reitermb ^^ . @lfrohlich The one you tagged could be Miles doppelganger!

Reply:
@lfrohlich Fixed! Found one more in there too

Reply:
oh no! thanks @reitermb